samples/run_jobs.py

```diff
@@ -20,32 +25,83 @@
 recipe = Recipe(owner, recipe_name, recipe_tag, client=api_client)
 recipe.add_to_project(f'{owner}/{project}')
 
-samples_path = Path(__file__).parent.resolve().joinpath('samples.json')
-with open(samples_path) as samples_json:
-    samples = json.load(samples_json)
-
-for idx, sample in enumerate(samples):
-    job_name = sample.get('name', f'Sample {idx}')
-    datetime_now = datetime.datetime.now().strftime("%Y-%m-%d %H:%M:%S")
-    auto_name = f'(Scheduled by GitHub workflow: {datetime_now})'
-    job_name = ' '.join([job_name, auto_name])
-    description = sample.get('description', None)
-    job = NewJob(
-        owner, project, recipe, name=job_name, description=description,
-        client=api_client
-    )
+# load recipe inputs for each sample run
+samples_path = Path(__file__).parent.resolve().joinpath('sample_runs.json')
+with open(samples_path, encoding='utf-8') as samples_json:
+    sample_runs = json.load(samples_json)
 
+# create a new job
+datetime_now = datetime.datetime.now().strftime("%Y-%m-%d %H:%M:%S")
+name = f'Samples (Scheduled by GitHub workflow: {datetime_now})'
+new_study = NewJob(owner, project, recipe, name=name, client=api_client)
+
+# get recipe inputs for each run and upload artifact
+study_inputs = []
+artifacts_paths = {}
+for sample_run in sample_runs:
     inputs = {}
-    for recipe_input, value in sample['inputs'].items():
+    for recipe_input, value in sample_run.items():
         input_path = Path(__file__).parent.resolve().joinpath(value)
         if input_path.exists():
-            artifact_path = job.upload_artifact(input_path, f'sample_{idx}')
-            inputs[recipe_input] = artifact_path
+            if artifacts_paths.get(input_path, None) is None:
+                # artifact not uploaded yet
+                artifact_path = new_study.upload_artifact(input_path)
+                inputs[recipe_input] = artifact_path
+                artifacts_paths[input_path] = artifact_path
+            else:
+                # artifact already uploaded, reuse artifact path
+                inputs[recipe_input] = artifacts_paths[input_path]
         else:
+            # not a file recipe input
             inputs[recipe_input] = value
+    study_inputs.append(inputs)
+
+# add the inputs to the study
+new_study.arguments = study_inputs
 
-    arguments = []
-    arguments.append(inputs)
-    job.arguments = arguments
+# create the study
+running_study = new_study.create()
```

Comment:
I suggest printing out the path to the study to make it easier to quickly check the study on Pollination. Particularly, if it fails.